x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -340,6 +345,11 @@ public SharedBlobCacheService(
         decayTask.rescheduleIfNecessary();
         this.rangeSize = SHARED_CACHE_RANGE_SIZE_SETTING.get(settings);
         this.recoveryRangeSize = SHARED_CACHE_RECOVERY_RANGE_SIZE_SETTING.get(settings);
+        this.evictedCountNonZeroFrequency = meter.registerLongCounter(
+            "elasticsearch.blob_cache.evicted_count_non_zero_frequency",
```

Comment:
Nit: I think the goal of this metric is to determine the number of used regions that have been evicted in order to measure cache churn for instance sizing.  Right now, we want to check if it's non-zero.  After prefetching, we'll want to check that it's greater than 1 (probably).

Maybe a name like `countOfEvictedUsedRegions` or something similar would be a bit more future proof?  That way we won't need to change the metric name if we change the way this is determined.

Reply:
Makes sense, I'll keep the snake_case for readability and to be aligned with other metrics.

Reply:
@original-brownbear I'm looking to confirm that frequency can be used as a proxy for "use". 

Can frequency reduce to zero after a time, regardless of usage? If so, maybe this should be reverted to the original name...  

I want to ensure that my suggestion to rename is correctly describing the behaviour. Thanks!